packages/web-app/src/pages/PersonEditPage/index.jsx

```diff
@@ -0,0 +1,285 @@
+      ],
+
+      {
+        shouldFocus: true
+      }
+    );
+    isValid.true();
+
+    if (result) {
+      setActiveStep(prevActiveStep => prevActiveStep + 1);
+    }
+  };
+
+  const handleBack = () => {
+    setActiveStep(prevActiveStep => prevActiveStep - 1);
+  };
+
+  const onSubmit = async data => {
+    setActiveStep(prevActiveStep => prevActiveStep + 1);
+
+    const User = makeUserData(data);
+
+    dispatch(updateUser(User));
+
+    if (data.user.email !== undefined && data.user.email !== '') {
+      dispatch(postChangeEmail(data.user.email));
+    }
+    if (data.user.password !== undefined && data.user.password !== '') {
+      dispatch(
+        postChangePassword(data.user.password, authState.authTokenDecoded)
+      );
+    }
+  };
+
+  const handleReset = () => {
+    reset({ user: defautValues });
+    setActiveStep(0);
+  };
+
+  const handleFinish = () => {
+    history.push(`/ui/persons/:${userValues.id}`);
+  };
+
+  return (
+    <Layout
+      title={formatMessage({ id: 'Edit your profile' })}
+      footer=""
+      content={
+        isSubmitted && isNil(UserError) ? (
+          <Box display="flex" justifyContent="center" flexDirection="column">
+            {UserLoading && (
+              <>
+                <Typography>
+                  {formatMessage({
+                    id: 'Updating user...'
+                  })}
+                </Typography>
+                <CircularProgress />
+              </>
+            )}
+            {!UserLoading && isSubmitSuccessful && (
+              <form>
+                <Alert
+                  severity="success"
+                  title={formatMessage({
+                    id: 'User successfully updated'
+                  })}
+                />
+              </form>
+            )}
+            {!UserLoading && !isSubmitSuccessful && (
+              <form>
+                <Alert
+                  severity="error"
+                  title={formatMessage({
+                    id: 'An error occurred when updating the user!'
+                  })}
+                />
+              </form>
+            )}
+          </Box>
+        ) : (
+          <Box display="flex" justifyContent="center" flexDirection="column">
+            <form autoComplete="off" onSubmit={handleSubmit(onSubmit)}>
+              <Stepper activeStep={activeStep}>
+                {steps.map(label => {
+                  const stepProps = {};
+                  const labelProps = {};
+
+                  return (
+                    <Step key={label} {...stepProps}>
+                      <StepLabel {...labelProps}>{label}</StepLabel>
+                    </Step>
+                  );
+                })}
+              </Stepper>
+
+              <div>
+                {activeStep === steps.length ? (
+                  <div>
+                    <Typography className={classes.instructions}>
+                      All steps completed - you&apos;re finished
```

Comment:
A traduire en utilisant "formatMessage". 

"You're finished" je ne crois pas que ça se dise en anglais, ou en tout cas, pas dans ce sens-là 😅